### ISSUE TITLE
Maximum call stack Exceeded

### ISSUE THREAD

**Issue by rosemalejohn** [leonardovilarinho/vue-acl#26]
I am using VueJS 2.5.2 and vue-router 2.7.0

**Comment by leonardovilarinho** [comment]
Hi,

add more information for me. Your router and main file, please.

**Comment by ilvalerione** [comment]
Hello Leonardo, I meet the same issue.

I have the HomePage route with free access permission:

```javascript
{
            path: '/',
            name: 'HomePage',
            component: require('../components/pages/HomePage.vue'),
            meta: {
                permission: 'any'
            }
},
```

And another route with more restrictive permissions:
```javascript
{
        path: '/rdp',
        name: 'rdp',
        component: require('./RDPListPage.vue'),
        meta: {
            permission: 'read-rdp'
        }
},
```

When I navigate from HomePage to RDP all works fine, but when I try to click on the HomePage link to return in the HomePage I receive "Maximum call stack Exceeded" exception how you can see below:

![immagine](https://user-images.githubusercontent.com/13559278/31992237-079ef75e-b97a-11e7-8fc5-05842327bc7e.png)

Before adding your ACL plugin all worked fine.

**Comment by ilvalerione** [comment]
As result of others test, I noticed that the exception is fired with all routes that have "any" permission in meta object!

**Comment by leonardovilarinho** [comment]
I working for it.

**Comment by leonardovilarinho** [comment]
I fix in 128e19c325c8b8057013c3fab2ad583394dcfe5b, please run update command.
